.github/workflows/automation-benchmark-tests.yml

```diff
@@ -63,24 +63,15 @@ jobs:
       TEST_INPUTS: ${{ inputs.TestInputs }}
       CHAINLINK_ENV_USER: ${{ github.actor }}
       REF_NAME: ${{ github.head_ref || github.ref_name }}
-      ENV_JOB_IMAGE_BASE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink-tests
     steps:
       - name: Setup Push Tag
-        id: push
         shell: bash
         run: |
           echo "### chainlink image used for this test run :link:" >>$GITHUB_STEP_SUMMARY
           echo "\`${{ inputs.chainlinkVersion }}\`" >>$GITHUB_STEP_SUMMARY
           echo "### chainlink-tests image tag for this test run :ship:" >>$GITHUB_STEP_SUMMARY
-          if [ "${{ env.REF_NAME }}" != "develop" ]; then
-            # use sha if not on develop
-            echo "tag=${{ env.ENV_JOB_IMAGE_BASE }}:benchmark.${{ github.sha }}" >>$GITHUB_OUTPUT
-            echo "\`benchmark.${{ github.sha }}\`" >>$GITHUB_STEP_SUMMARY
-          else
-            # default to develop
-            echo "tag=${{ env.ENV_JOB_IMAGE_BASE }}:develop" >>$GITHUB_OUTPUT
-            echo "\`develop\`" >>$GITHUB_STEP_SUMMARY
-          fi
+          echo "\`${{ github.sha }}\`" >>$GITHUB_STEP_SUMMARY
```

Comment:
```suggestion
          echo "${GITHUB_SHA}" >> $GITHUB_STEP_SUMMARY
```

Reply:
Is using ${{GITHUB_SHA}} over ${{github.sha}} just the personal preference we are going with or is there a legitimate reason?

Also to note default shell formatting (based on shellcheck and shfmt) doesn't have a space between the >> and the $GITHUB_STEP_SUMMARY. Both will actually flag that if you add the space.

Reply:
> Is using ${{GITHUB_SHA}} over ${{github.sha}} just the personal preference we are going with or is there a legitimate reason?

I'd say it's less error prone. No need to worry about escaping the GitHub context var. Not a huge deal but IMO it's cleaner.

> Also to note default shell formatting (based on shellcheck and shfmt) doesn't have a space between the >> and the $GITHUB_STEP_SUMMARY. Both will actually flag that if you add the space.

I just tested shellcheck and it seems like it's happy with the space and without the space between the redirect operator. Not a huge deal either way.